boards/nordic/nrf5340_audio_dk/nrf5340_audio_dk_nrf5340_shared.dtsi

```diff
@@ -70,6 +70,35 @@
 		};
 	};
 
+	arduino_header: connector {
```

Comment:
Please delete the definition of this node from https://github.com/zephyrproject-rtos/zephyr/blob/0834432e29f8d5d4518f44c06ad5e8abc6e5d85a/boards/nordic/nrf5340_audio_dk/nrf5340_audio_dk_nrf5340_cpuapp_common.dtsi#L28
There is no need to define it twice for cpuapp[_ns].